docs/operations/dynamic-graphs.md

```diff
@@ -91,26 +91,6 @@ traversal references, you must:
 1.  Configure each node to use the
     [ConfiguredGraphFactory](configured-graph-factory.md#configuring-JanusGraph-server-for-configuredgraphfactory).
 
-2.  Configure each node to use a `JanusGraphChannelizer`, which injects
-    lower-level Gremlin Server components, like the GremlinExecutor,
-    into the JanusGraph project, giving us greater control of the
-    Gremlin Server.
-
-To configure each node to use a `JanusGraphChannelizer`, we must update
-the `gremlin-server.yaml` to do so:
-
-    channelizer: org.janusgraph.channelizers.JanusGraphWebSocketChannelizer
-
-There are a few channelizers you can choose from:
-
-1.  org.janusgraph.channelizers.JanusGraphWebSocketChannelizer
-2.  org.janusgraph.channelizers.JanusGraphHttpChannelizer
-3.  org.janusgraph.channelizers.JanusGraphNioChannelizer
-4.  org.janusgraph.channelizers.JanusGraphWsAndHttpChannelizer
-
-All of the channelizers share the exact same functionality as their
-TinkerPop counterparts.
-
```

Comment:
Why do we need to use Tinkerpop channelizer's? The documentation says websocket is the default case: https://tinkerpop.apache.org/docs/3.4.10/reference/#_connecting_via_http

Reply:
I was thinking that it could be helpful for users to know about a default behavior and possible configurations. That said, I'm fine removing it. The only thing is that `Set Up` section now contains only 1 step. In such case it makes sense to replace it with something like this:
`To set up your cluster to bind dynamically created graphs and their traversal references, you must configure each node to use the ConfiguredGraphFactory.`